src/index.json

```diff
@@ -972,7 +972,7 @@
                     "metadata_version": "2.0",
                     "name": "find",
                     "summary": "Intelligent querying for CLI information.",
-                    "version": "0.1.0"
+                    "version": "0.2.0"
```

Comment:
How do you want to differentiate between a Minor version and a Patch version?

Reply:
I thought this is minor version as the experience has changed more than a patch.